src/main/java/lsd/format/json/ByteArraySerializer.kt

```diff
@@ -0,0 +1,36 @@
+package lsd.format.json
+
+import com.fasterxml.jackson.core.JsonGenerator
+import com.fasterxml.jackson.databind.SerializerProvider
+import com.fasterxml.jackson.databind.ser.std.StdSerializer
+import lsd.format.config.log
+import lsd.format.xml.indentXml
+import java.io.IOException
+
+/**
+ * Replaces the default byte[] serializer so that we can pretty print json strings that may be contained inside the
+ * bytes[] when converted to a String object.
+ */
+class ByteArraySerializer @JvmOverloads constructor(t: Class<ByteArray?>? = null) : StdSerializer<ByteArray>(t) {
```

Comment:
I think this file can be removed now since we handle ByteArrays explicitly now by converting them to Strings.

What do you think @nickmcdowall ?

Reply:
yep sounds logical to remove in that case 👍

Reply:
turns out, it's still needed when the object we serialise has a ByteArray field... 😄